FerramAerospaceResearch/LEGACYferram4/FARControllableSurface.cs

```diff
@@ -571,9 +624,21 @@ private void AoAOffsetFromControl()
             }
 
             AoAdesiredControl *= AoAsign;
+            AoAdesiredControl *= CalculateDynamicControlFactor();
             AoAdesiredControl = AoAdesiredControl.Clamp(-Math.Abs(maxdeflect), Math.Abs(maxdeflect));
         }
 
+        private double CalculateDynamicControlFactor()
+        {
+            double factor = 1;
+            if (vessel.atmDensity > 0.01 && isDynamicDeflection)
+            {
+                 factor = Math.Pow(dynamicControlStartSpeed, exponent) / Math.Pow(vessel.srfSpeed, exponent) / (vessel.atmDensity / vessel.lastBody.atmDensityASL);
```

Comment:
The two `Mtath.Pow` should be combined into `Math.Pow(dynamicControlStartSpeed / vessel.srfSpeed, exponent)`